src/bin/traditional.rs

```diff
@@ -45,6 +49,85 @@ fn hi(_: PotatoRequest) -> PotatoResponse {
         .add_header("Content-Length", &body.len().to_string())
 }
 
+fn simple_add(req: PotatoRequest) -> PotatoResponse {
+    let res = PotatoResponse::new();
+
+    let num_str = req.body.unwrap();
+    let num_str = str::from_utf8(&num_str).unwrap();
```

Comment:
```suggestion
    let num_str = String::from_utf8_lossy(&num_str);
```
Use String's lossy so there is no unwrap